search/indexing_api.py

```diff
@@ -109,6 +109,9 @@
     "image_src": {"type": "keyword"},
     "topics": {"type": "keyword"},
     "offered_by": {"type": "keyword"},
+    "created_on": {"type": "date"},
```

Comment:
It looks like we are naming a similar field `created` on `CONTENT_OBJECT_TYPE` above. Can we rename this to match that field for consistency?